tests/test_climatology_ESP_verfication.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Feb 23 22:25:32 2021
+
+@author: ets
+"""
+
+import datetime as dt
+
+from ravenpy.utilities.testdata import get_local_testdata
+from ravenpy.utilities.forecasting import make_ESP_hindcast_dataset, make_climpred_hindcast_object
+
+class TestClimpredHindcastVerification:
```

Comment:
This test takes about 65 seconds to run on my end. I suggest we use the minimum forecast duration and number of years to get a valid test (in terms of software, not hydrology) to speed up the test.

Reply:
I've shortened the period and forecast duration so that it is a super small test, takes 11 seconds now on my VM. Can't go faster unless we change the ESP dataset, and as far as I know we don't have another one like this lying around (as we use all years present in the dataset for now).

Agreed with all the rest (i.e. optimizations in the maintenance phase).

Can we merge now or is there still a problem with versioning as mentioned by @Zeitsperre with #6?

Reply:
The issue is that, as I understand it, `climpred` support is dependent on #6. If that isn't the case, I can add it to the setup requirements on this branch.

Reply:
OK! @huard can confirm but I did not change anything nor pull the #6 PR branch when I did my setup, so I think it might work anyways?

Reply:
Alright. In that case, I can port my changes from there to here.

Reply:
Done!